services/gateway-api/src/circuit_breaker/circuit_breaker.py

```
@@ -0,0 +1,463 @@
+
+        Usage:
+            @breaker.protected
+            async def my_api_call():
+                return await some_api()
+        """
+        @wraps(func)
+        async def wrapper(*args, **kwargs):
+            return await self.call(func, *args, **kwargs)
+        return wrapper
+
+    async def __aenter__(self):
```

Ruff (0.14.8)

Missing return type annotation for special method `__aenter__`

(ANN204)